force-app/main/default/triggers/OpportunityTrigger.trigger

```diff
@@ -15,26 +15,30 @@ For this lesson, students have two options:
 
 Remember, whichever option you choose, ensure that the trigger is activated and tested to validate its functionality.
 */
-trigger OpportunityTrigger on Opportunity (before update, after update, before delete) {
+trigger OpportunityTrigger on Opportunity (before update, before delete, after insert, after update, after delete, after undelete) {
 
+    new OpportunityTriggerHandler().run();
```

Comment:
This is good but I am confused about the other opportunity Triggers

Reply:
As the HW itsself is provided with two different triggers on Opportunity object, I implemented the logic separately in their own triggers. But as a best practive I'll use only one Trigger per object going ahead.